app/onboarding/views/account_activated.py

```diff
@@ -0,0 +1,41 @@
+from app.onboarding.base import onboarding_bp
+from enum import Enum
+from flask import redirect, render_template, request, url_for
+
+
+CHROME_EXTENSION_LINK = "https://chrome.google.com/webstore/detail/simpleloginreceive-send-e/dphilobhebphkdjbpfohgikllaljmgbn"
+FIREFOX_EXTENSION_LINK = "https://addons.mozilla.org/firefox/addon/simplelogin/"
+
+
+class Browser(Enum):
+    Firefox = 1
+    Chrome = 2
+    Other = 3
+
+
+def get_browser() -> Browser:
+    user_agent = request.user_agent
+    if user_agent.browser in ["chrome", "edge", "opera", "webkit"]:
+        return Browser.Chrome
+    elif user_agent.browser in ["mozilla", "firefox"]:
+        return Browser.Firefox
+    return Browser.Other
+
+
+@onboarding_bp.route("/account_activated", methods=["GET"])
+def account_activated():
```

Comment:
should we mark this route for logged user only?

Reply:
If a user is able to reach this page without being logged in, nothing would happen.

If they clicked on the "Install extension" button, nothing would break. Also, if they clicked on the "Go to web app" they would be prompted to log in anyways.

Reply:
IT's a bit weird that an unauth user can see the activated page. Better to add the decorator and this way they will be sent to login.